README.md

```diff
@@ -1 +1,38 @@
-# hello-microverse1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse!
+
+> Description the project.
```

Comment:
- [ ] [OPTIONAL] Kindly edit the project description to better suit your project.